mlflow/server/js/src/app.tsx

```diff
@@ -15,6 +15,9 @@ import { MlflowRouter } from './MlflowRouter';
 
 export function MLFlowRoot() {
   const i18n = useI18nInit();
+  const [isDarkTheme, setIsDarkTheme] = React.useState(
+    window.matchMedia('(prefers-color-scheme: dark)').matches || false,
```

Comment:
@harupy does this seem like a reasonable default or are there better ways to get the preferred scheme from the browser?

Reply:
Is `prefers-color-scheme` like the system preferece?

Reply:
looks like it: https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme